cmake/HandlePython.cmake

```diff
@@ -1,22 +1,48 @@
 # Set Python version if either Python or MATLAB wrapper is requested.
 if(GTSAM_BUILD_PYTHON OR GTSAM_INSTALL_MATLAB_TOOLBOX)
-    if(${GTSAM_PYTHON_VERSION} STREQUAL "Default")
-        # Get info about the Python3 interpreter
-        # https://cmake.org/cmake/help/latest/module/FindPython3.html#module:FindPython3
-        find_package(Python3 COMPONENTS Interpreter Development)
+  if(${GTSAM_PYTHON_VERSION} STREQUAL "Default")
 
-        if(NOT ${Python3_FOUND})
-            message(FATAL_ERROR "Cannot find Python3 interpreter. Please install Python >= 3.6.")
-        endif()
+    if(${CMAKE_VERSION} VERSION_LESS "3.12.0")
```

Comment:
Did you test this with 3.9 and 3.12? If so ignore :)

Reply:
I tested it with 3.10 which is what comes standard in Ubuntu 20.04.

Reply:
Great